src/js/polyfills/webvtt.js

```diff
@@ -454,19 +455,22 @@ function CueStyleBox(window, cue) {
     // https://w3c.github.io/webvtt/#ref-for-enumdef-positionalignsetting-1
     // The cue.align property is settable and other browsers use it as the offset from which the cue.position
     // value is applied.
+    let transform = '';
```

Comment:
transform ensures that we respect the cue alignment when determining position. now that these are inline-block elements the size is not known at time of instantiation. this new code adds a transform to properly place the cue based on alignment and then removes it when the stylebox is finally moved to prevent it from being applied twice (once during exact px calculations and then after when we go to display)